theme.ts

```diff
@@ -0,0 +1,20 @@
+import { DefaultTheme } from 'styled-components';
```

Comment:
this theme is fine, but you have to add all the colors inside it and use it through the app